docs/developer/releasing.md

```diff
@@ -62,30 +62,39 @@ export GPG_TTY=$(tty)
 
 ## Performing Releases
 
-The lifetime of a release branch is sheparded by a single person, picked from a
-rota of project maintainers and contributors. The release shepard is
+The lifetime of a release branch is shepherded by a single person, picked from a
+rota of project maintainers and contributors. The release shepherd is
 responsible for managing all releases that compose the release branch.
 
-For a new release branch, the release shepard will:
+For a **new release candidate**, the release shepherd will:
 
 1. Gather consensus on which commit should be used as a base for the release
    branch.
 
-2. Create and push the release branch from the selected base commit.
+2. Create and push the release branch from the selected base commit:
+
+   * The name of the release branch should be the name of the stable version we intend 
+   to release, e.g. `release-v0.31`.
+   * Release branches do not contain the `-rc.N` release candidate suffix. I.e. there 
+   is no branch called `release-v0.31-rc.0`.
+   * Release branches do not contain the patch version. I.e. there is no branch called 
+   `release-v0.31.0` or `release-v0.31.4`.
```

Comment:
The Google documentation style guide recommends against using e.g. and i.e. since people tend to mix them up.

```suggestion
   * The name of the release branch should be the name of the stable version we intend 
   to release, such as `release-v0.31`.
   * Release branches do not contain the `-rc.N` release candidate suffix. This means
     there is no branch called `release-v0.31-rc.0`.
   * Release branches do not contain the patch version. This means there is no 
     branch called `release-v0.31.0` or `release-v0.31.4`.
```